cmd/cluster/support/post.go

```diff
@@ -144,11 +146,11 @@ func (o *postOptions) run() error {
 
 // createPostRequest create and populates the limited support post call
 // swagger code gen: https://api.openshift.com/?urls.primaryName=Clusters%20management%20service#/default/post_api_clusters_mgmt_v1_clusters__cluster_id__limited_support_reasons
-func createPostRequest(ocmClient *sdk.Connection, cluster *v1.Cluster) (request *sdk.Request, err error) {
+func createPostRequest(client SDKConnectionClient, clusterID string) (request *sdk.Request, err error) {
```

Comment:
Same questions here as above; why change to client, and does this work as expected with external IDs?